### ISSUE TITLE
Add a formatter to the CI/CD pipeline

### ISSUE THREAD

**Issue by j4v** [nccgroup/ScoutSuite#645]
**Is your feature request related to a problem? Please describe.**

Add a formatter to the CI/CD pipeline, to enforce/automate following PEP8.

**Describe the solution you'd like**

There are a few options:
- https://github.com/hhatto/autopep8
- https://github.com/psf/black
  - doesn't support python 3.5
- ?

**Describe alternatives you've considered**

Hope for people to follow PEP8.

**Comment by j4v** [comment]
Closing as implemented by https://github.com/nccgroup/ScoutSuite/pull/646.

**Comment by j4v** [comment]
@klauern does https://github.com/nccgroup/ScoutSuite/pull/646 actually do anything? I've never seen it modify the code.

Also just checking that it's still present in https://github.com/nccgroup/ScoutSuite/pull/783?

**Comment by klauern** [comment]
Yes, it's included still as part of #783: https://github.com/nccgroup/ScoutSuite/pull/783/files#diff-913fedaed3cb05fe362c0ceb35e2d827R5

I'm not sure if #646 is still relevant, tbh.  It has been sitting around for a while and with #783 it might not matter as much.

**Comment by j4v** [comment]
Reopening this issue.

Now that we're getting rid of python 3.5 support, we'll re-introduce Black at the CI/CD level and get rid of LGTM since it's noisy and haven't added much value.

**Comment by j4v** [comment]
Could also run the following on each run:
- https://github.com/nccgroup/ScoutSuite/blob/master/tools/sort-ruleset.py
- https://github.com/nccgroup/ScoutSuite/blob/master/tools/format_findings.py
